style.css

```diff
@@ -551,9 +636,199 @@ button.interaction:disabled {
   background-size: 18px;
 }
 
+@media only screen and (min-width: 768px) {
+  main {
+    display: grid;
+    grid-template-rows: 0.1fr 1fr 0.83fr;
+    grid-template-columns: 1fr;
+    gap: 0;
+    width: 100%;
+    height: fit-content;
+    padding: 0;
+  }
+
+  .title {
+    margin-top: 0;
+    border: none;
+    grid-row: 1 / 2;
+    grid-column: 1 / 2;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-start;
+  }
+
+  .h3-title {
+    margin-left: 6%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  hr {
+    visibility: visible;
+    height: 1px;
+    border-width: 0;
+    background-color: #212121;
+    width: 52%;
+    margin-right: 15.5%;
+  }
+
+  .frame44 {
+    height: fit-content;
+    grid-row: 2 / 3;
+    grid-column: 1 /2;
+    width: 100%;
+    padding-bottom: 0;
+  }
+
+  .featured-img {
+    width: 90%;
+    height: fit-content;
+    display: grid;
+    grid-template-rows: 0.4fr 1fr;
+    grid-template-columns: 1fr 0.8fr;
+    margin-left: 0.5%;
+    padding-left: 0;
+  }
+
+  .featured-img > img {
+    grid-row: 1 / 2;
+    grid-column: 1 / 2;
+    width: 100%;
+    height: fit-content;
+    align-self: start;
+  }
+
+  .right-block {
+    grid-row: 1 / 2;
+    grid-column: 2 / 3;
+    width: 100%;
+    height: fit-content;
+    gap: 30px;
+  }
```

Comment:
@jerryowusu  Kindly suggest to me how I can do that. I had thought of using fixed measurements in width and height but I don't think that will make it look good in all screen sizes above 768px that's why I used percentages. Any suggestions on how to solve this??

Reply:
> @jerryowusu Kindly suggest to me how I can do that. I had thought of using fixed measurements in width and height but I don't think that will make it look good in all screen sizes above 768px that's why I used percentages. Any suggestions on how to solve this??

@tobuya  You can also consider using vh or vw units since that is more responsive and flexible. Also, you can add more than one media breakpoint. For instance, you can create a breakpoint at 768px, 900px, 1200px, 1400px etc and tweak the dimensions to fix it.